module/localStorage.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line
+import Booklibrary from './books.js';
+
+export default class Books {
+    booklibrary = new Booklibrary();
+
+  static books = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+
+  static add(author, title) {
+    const book = { title, author };
+    Books.books.push(book);
+    localStorage.setItem('books', JSON.stringify(Books.books));
+    Books.display();
+  }
+
+  static remove(author, title) {
+    for (let i = 0; i < Books.books.length; i += 1) {
+      if (Books.books[i].title === title && Books.books[i].author === author) {
+        Books.books.splice(i, 1);
+      }
+    }
+    //   localStorage.clear();
+    localStorage.setItem('books', JSON.stringify(Books.books));
+    Books.display();
+  }
```

Comment:
- [ ] Your website allows users to add books, which are then stored in the local storage. Great work👏. However, I am still unable to remove some books from the books list after clicking on the remove button. Please make sure the user can remove any book and that the local storage is updated once any book is removed. 

Hint: I recommend using the filter() method to remove a book from an array of books since it doesn't mutate the array.
```js
books = books.filter(({ id }) => bookID !== id);
```